src/cite.cpp

```diff
@@ -79,14 +79,14 @@ void CitationManager::insert(const QCString &label)
 {
   p->entries.insert(
       std::make_pair(
-        label.str(),
-        std::make_unique<CiteInfoImpl>(label)
+        label.lower().str(),
+        std::make_unique<CiteInfoImpl>(label.lower())
       ));
 }
 
 const CiteInfo *CitationManager::find(const QCString &label) const
 {
-  auto it = p->entries.find(label.str());
+  auto it = p->entries.find(label.lower().str());
```

Comment:
I think the `.lower()` calls below can be avoided by using `find()` and `insert()` everywhere instead of `p->entries.find()` and `p->entries.insert()`.